### PR TITLE
Labels needed for Label Freeze

### PR DESCRIPTION
**Pull request by meena-sfdc**
Add new custom labels, new custom fields and their updates to master before translations label freeze.

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata
Custom Label:
- AssistiveTextInfo
- RD2_EnablementDryRunOutdatedError
- RD2_EnablementDryRunOutdatedInfo
- RD2_InstallmentFrequencyMustBeValid
- RD2_NumberInstallmentsCannotBeLessThanPaid
- RD2_NumberInstallmentsNotAllowedWhenOpen
- RD2_NumberInstallmentsRequiredWhenFixed
- RD_ErrorAddDonationLimitedToLegacyMode

Custom Field:
- npe03__Recurring_Donations_Settings__c.NextDonationDateMatchRangeDays__c
- npe03__Recurring_Donations_Settings__c.UseFiscalYearForRecurringDonationValue__c

# Deleted Metadata

### CODE REVIEW

**Comment by meena-sfdc** [comment]
**lurch: attach W-039320

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001Hs3fQAC" target="_blank">W-039320</a>

File: src/objects/npe03__Recurring_Donations_Settings__c.object [hunk 1]
```diff
@@ -68,6 +68,20 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>NumberOfDaysFromNextDonationDate__c</fullName>
+        <defaultValue>0</defaultValue>
```
Comment (force2b) [inline src/objects/npe03__Recurring_Donations_Settings__c.object]: As per discussion on W-038890, this new field should have a default value of 3.

Comment (RadSFDeveloper) [reply src/objects/npe03__Recurring_Donations_Settings__c.object]: The API name has been changed to `NextDonationDateMatchRangeDays__c`. 
@sam-knox , can you please advise if the field label and description/help text should be changed?

**Comment by meena-sfdc** [comment]
@force2b , The changes are done and ready for review!

File: src/objects/npe03__Recurring_Donations_Settings__c.object [hunk 2]
```diff
@@ -68,6 +68,20 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>NextDonationDateMatchRangeDays__c</fullName>
```
Comment (RadSFDeveloper) [inline src/objects/npe03__Recurring_Donations_Settings__c.object]: Replace `'` with `&apos;`.

File: src/objects/npe03__Recurring_Donation__c.object [hunk 3]
```diff
@@ -35,6 +35,11 @@
                     <default>false</default>
                     <label>Card Expired</label>
                 </value>
+                <value>
```
Comment (RadSFDeveloper) [inline src/objects/npe03__Recurring_Donation__c.object]: There is an error with the Description/Help Text for the Current and Next Year Value fields. Can you please replace `&quot;` with `&apos;` ?

Comment (meena-sfdc) [reply src/objects/npe03__Recurring_Donation__c.object]: Will change it , @RadSFDeveloper ! In this PR, there is pipeline character in custom field help text for UseFiscalYearForRecurringDonationValue__c, does it needs to be replaced or its fine?

Comment (RadSFDeveloper) [reply src/objects/npe03__Recurring_Donation__c.object]: That one is fine, thanks for checking @meena-sfdc .

File: src/labels/CustomLabels.labels [hunk 4]
```diff
@@ -1896,6 +1928,30 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
...
+        <shortDescription>Disallow a value in Planned Installments on the RD for Recurring Type is Open</shortDescription>
+        <value>Number of Planned Installments is for Fixed Recurring Donations only. Remove the value in Number of Planned Installments or change the Recurring Type to Fixed.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_NumberPlannedInstallmentsCannotBeLessThanPaid</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Validate Planned Installments field is greater than number of Paid Installments</shortDescription>
+        <value>Number of Planned Installments can't be less than the Number of Paid Installments.</value>
```
[the remark this answers is not in the document]
Comment (force2b) [reply src/labels/CustomLabels.labels]: @meena-sfdc As per the other PR, this label has been changed to `RD2_NumberInstallmentsCannotBeLessThanPaid`

**Comment by meena-sfdc** [comment]
@force2b and @RadSFDeveloper , The changes are done and ready for review.

**Comment by sam-knox** [comment]
@meena-sfdc @RadSFDeveloper  - Based on the discussion above, how about this for a label?

"Next Donation Date Match Range"

I think the help text can stay the same.

**Comment by force2b** [comment]
I like that @sam-knox!

**Comment by meena-sfdc** [comment]
@heidmary , This PR is ready for testing!

**Pull request merged**